x-pack/plugins/profiling/public/components/topn_functions/index.tsx

```diff
@@ -189,31 +202,43 @@ export function TopNFunctionsTable({
             })
           : undefined;
 
-      const diff =
-        comparisonTopNFunctions && comparisonRow
-          ? {
-              rank: topN.Rank - comparisonRow.Rank,
-              samples: topN.CountExclusive - comparisonRow.CountExclusive,
-              exclusiveCPU:
-                exclusiveCPU -
-                (comparisonRow.CountExclusive / comparisonTopNFunctions.TotalCount) * 100,
-              inclusiveCPU:
-                inclusiveCPU -
-                (comparisonRow.CountInclusive / comparisonTopNFunctions.TotalCount) * 100,
-            }
-          : undefined;
+      function calculateDiff() {
+        if (comparisonTopNFunctions && comparisonRow) {
+          const comparisonCountExclusiveScaled = scaleValue({
+            value: comparisonRow.CountExclusive,
+            scaleFactor: comparisonScaleFactor,
+          });
+
+          return {
+            rank: topN.Rank - comparisonRow.Rank,
+            samples: topNCountExclusiveScaled - comparisonCountExclusiveScaled,
+            exclusiveCPU:
+              exclusiveCPU -
+              (comparisonRow.CountExclusive / comparisonTopNFunctions.TotalCount) * 100,
+            inclusiveCPU:
+              inclusiveCPU -
+              (comparisonRow.CountInclusive / comparisonTopNFunctions.TotalCount) * 100,
```

Comment:
Question: I'm not sure if I should scale the CPU values too! @elastic/profiling-ui does anybody know it?

Reply:
TIL 
wow, great explanation! thank you! 🤓